test_sets/SDL5_0/cancel_interaction.txt

```diff
@@ -0,0 +1,7 @@
+./test_scripts/API/CancelInteraction/001_SendCancelInteraction_PerformInteraction_HMIReceivesRequest.lua
```

Comment:
This should be moved to `test_sets/cancel_interaction.txt`

Reply:
moved in https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/2206/commits/d85311ff05a3e1d376c974df089d4949ad759edc